src/components/commandHandler.ts

```diff
@@ -24,7 +24,7 @@ import { playbackManager } from './playbackManager';
 import { DocumentManager } from './documentManager';
 
 export abstract class CommandHandler {
-    private static playerManager: cast.framework.PlayerManager;
+    private static playerManager: framework.PlayerManager;
```

Comment:
just a question here, I don't fully understand why "cast" is implicit here..

Reply:
Found [this](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/28190#issuecomment-414637374) issue, basically the typings are not exported in the index, but in framework.

Shouldn't affect runtime though as it's all TypeScript abstraction as usual :)

Good catch!